jv-oop-advanced/src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+    private final int radius;
+
+    public Circle(Form randomForm, Color randomColor, int radius) {
+        super(randomForm,randomColor);
```

Comment:
```suggestion
        super(randomForm, randomColor);
```

Reply:
I need to do one space? he is already there .. maybe mistake on GitHub